src/main/java/com/example/report/service/ReportService.java

```diff
@@ -85,10 +83,10 @@ private Map<String, BigDecimal> calculator(BigDecimal amount, Employee employee,
             calculatedNumbers.put(GROSS_AMOUNT, amount.divide(BigDecimal.valueOf(0.8), RoundingMode.HALF_UP));
             return calculatedNumbers;
```

Comment:
We don't have to have `return` statement in each `if`, we can return the result once at the end